### PR TITLE
profile name as title & Favicons

### LINKED ISSUE #152: Show Profile Name as Title
In the head>title and in the Navbar as Title

### PR DESCRIPTION
**Pull request by RealSocius**
- Changed Page Titles
- Added favicon to navbar-brand

Resolves #152

### CODE REVIEW

**Review by JM-Lemmi — CHANGES_REQUESTED** [review]
ical-relay is supposed to stay generalized, so I'd like "Calendar" to not be Replaced by "Vorlesungsplan". But we can add a configuration option for "Name", so it can show up as Vorlesungsplan after configuration.

File: templates/head.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<link rel="shortcut icon" href="{{.FaviconPath}}" type="image/svg">
```
Comment (JM-Lemmi) [inline templates/head.html:3]: Do we need logic to differentiate between svg and png favicons?

**Comment by robske110** [comment]
I don't really think the faviconPath variable is needed given the path is going to be inside the templates folder anyway

**Comment by JM-Lemmi** [comment]
I didn't think we'd deliver the Favicon in the static directory, thus the configurability. If you change the favicon in the static directory it will get overwritten on the next update.

**Comment by robske110** [comment]
If we don't have it there I think it still makes sense to drop the configuration option and have it at a fixed path (maybe `config-path/favicon.svg` or `storage-path/favicon.svg`)

**Comment by JM-Lemmi** [comment]
Having the config path allows linking the favicon via URL too, allowing to use a parent sites favicon.

**Review by JM-Lemmi — APPROVED** [review]

File: config.yml.example [hunk 2]
```diff
@@ -3,6 +3,8 @@ server:
   loglevel: "info"
   url: "https://cal.julian-lemmerich.de"
   templatepath: /opt/ical-relay/templates
+  faviconpath: /static/media/favicon.svg
+  name: "Vorlesungsplan"
```
Comment (robske110) [inline config.yml.example:7]:
```suggestion
  name: "Calendar"
```
keep config.yml.example default in line with binary defaults

**Review by robske110 — CHANGES_REQUESTED** [review]

**Review by robske110 — APPROVED** [review]

**Pull request merged**